content-src/components/DiscoveryStreamBase/DiscoveryStreamBase.jsx

```diff
@@ -0,0 +1,24 @@
+import {connect} from "react-redux";
+import React from "react";
+
+export class _DiscoveryStreamBase extends React.PureComponent {
+  render() {
+    return (
+      <div className="discovery-stream">
+        <div className="layout">
+          {this.props.DiscoveryStream.layout.map((section, sectionIndex) => (
```

Comment:
Just to make sure, the old behavior rendered old layout for `[]` but here it'll end up rendering empty div.

Reply:
I will file a ticket to handle the empty state for this better

Reply:
Filed: https://bugzilla.mozilla.org/show_bug.cgi?id=1516034